tests/BenchmarkDotNet.Tests/Reports/ScaledPrecisionTests.cs

```diff
@@ -0,0 +1,108 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using BenchmarkDotNet.Attributes;
+using BenchmarkDotNet.Attributes.Jobs;
+using BenchmarkDotNet.Columns;
+using BenchmarkDotNet.Configs;
+using BenchmarkDotNet.Engines;
+using BenchmarkDotNet.Exporters;
+using BenchmarkDotNet.Loggers;
+using BenchmarkDotNet.Reports;
+using BenchmarkDotNet.Running;
+using BenchmarkDotNet.Tests.Mocks;
+using BenchmarkDotNet.Toolchains;
+using BenchmarkDotNet.Toolchains.Results;
+using BenchmarkDotNet.Validators;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace BenchmarkDotNet.Tests.Reports
+{
+    public class ScaledPrecisionTests
+    {
+        private readonly ITestOutputHelper output;
+
+        public ScaledPrecisionTests(ITestOutputHelper output)
+        {
+            this.output = output;
+        }
+
+        [Theory]
+        [InlineData(140)]
+        [InlineData(50)]
```

Comment:
`0` is also a valid baseline. Please, add this test and make sure that everything works fine.
Also, we need a test with 3 benchmark methods: a baseline, a method with scaled value > 0.01, and a method with scaled value < 0.01.

Reply:
If the baseline is 0, would you expect all Scaled columns to be "?" ?

Reply:
Yes, it's the expected output in the current version. The main thing that we should check: the new code doesn't throw any exception in such cases. It will help to refactor this logic in the future.